### DISCUSSION TITLE
【错误反馈】第05课代码中的relax引用问题

### DISCUSSION THREAD

**Discussion by Relph1119** [mlc-ai/mlc-zh#74] [Q&A] [answered]
**使用环境：**
Python 3.7.9
mlc-ai-nightly 0.9.dev1663+gc73f34568

**问题反馈：**
在第05课的Jupyter Notebook的代码中，relax需要补充引用，教案中可能忘记了指明，导入环境中可添加如下代码：
```python
from tvm.script.parser import relax
```

**Comment by Hzfengsy** [comment, marked as answer]
感谢反馈。能否提供更详细的信息来帮助我们改进。

从我个人的角度来看，我们的jupyter和[notes](https://mlc.ai/zh)都是经过执行的（尤其是notes是CI自动编译的），没有看到明显错误。可能我们miss了某些细节，麻烦指出，多谢！

**Reply by lizexu123** [reply]
TVM的官网是不是没有这个relax，我没法导入，同样显示了这个问题

**Reply by Relph1119** [reply]
该问题已解决，教程中导入的是tvm.relax
